### PR TITLE
Frontier - Update BLOCK to be after MARBLE

### PR DESCRIPTION
**Pull request by Marble-Block**

### CODE REVIEW

**Comment by Marble-Block** [comment]
Also, added external incentives.

File: src/config.ts [hunk 1]
```diff
@@ -76,6 +76,7 @@ export const LockupAbledPoolIds: {
 	'670': true,
 	'678': true,
 	'679': true,
+	'691': true,
```
Comment (JeremyParish69) [inline src/config.ts:79]: Can we please put the External incentive gauge in a different PR?

Comment (Marble-Block) [reply src/config.ts]: Sure thing, removed externals from this PR.

**Comment by JeremyParish69** [comment]
Block will not be moved up. External Incentives can be added in a new PR
